### DISCUSSION TITLE
copilot just wont suggest anything

### DISCUSSION THREAD

**Discussion by asmarly** [community/community#13481] [Copilot Conversations]
i've typed in the code
```js
function calculateDaysBetweenDates(begin, end) {
```
and it wont work, any ways of fixing this? im on VS code by the way

**Comment by nmcc1212** [comment]
what happens if you press control+enter?

edit: im on mac, idk if it is a different key on pc

**Reply by asmarly** [reply]
if i do ctrl enter it makes a new line

**Comment by SandeepKanekal** [comment]
Try using the key-bind `alt+\`, this is supposed to trigger the inline suggestion. Also check if copilot is enabled, it is the small tab in the bottom-right corner with the copilot icon. If you have changed the key-bind for triggering inline suggestions, use the respective key-bind.

**Reply by nmcc1212** [reply]
<img width="1019" alt="Screenshot 2022-03-25 at 11 04 59 am" src="https://user-images.githubusercontent.com/48334675/160109554-5b90c287-071d-41e8-be39-4d4dc9f53184.png">
it seems to be ⌥+\

**Reply by SandeepKanekal** [reply]
In MacOS, it is option. Whereas, it is alt in windows.

**Reply by nmcc1212** [reply]
i was more referring to `\` instead of `/`

**Reply by SandeepKanekal** [reply]
I'll edit my answer! Thanks for pointing it out.

**Reply by asmarly** [reply]
i'll try this when i get onto my pc
